.github/workflows/lint.yml

```diff
@@ -14,16 +14,14 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18.x
+          go-version: 1.19.x
 
       - name: Checkout code
         uses: actions/checkout@v3
 
       - name: Lint
         uses: golangci/golangci-lint-action@v3
         with:
-          # Version is fixated to 1.47.3 because of https://github.com/golangci/golangci-lint-action/issues/535
-          # Until issue is fixed or repo go version is bumped to 1.19
-          version: v1.47.3
+          version: latest
```

Comment:
I already had experience with this on other repo. One day, all the ongoing PRs are red without any change on the PR itself.

I prefer that we use a fixed version, which can skip the "patch" part of semver.
Or to use another way to manage the "latest" more dynamically, I.e., with a dynamic script calling a static file; an organization-wide configuration that can be updated from another repo, etc.

Reply:
I know exactly what you mean 🙂 

`golangci-lint` releases just bundle different linter versions, which have their own releases. These releases are of course not without possible bugs, and we'd be stuck until the linter maintainer pushes out a fix, and `golangci-lint` publishes a release containing it.

I've fixated the version to the latest one as of today: `v1.51`:

- [Fixated version](https://github.com/gnolang/gno/pull/536/commits/ab5131f1b223582331be7e7b90d6f57ee561bf0b)
- [Dropped the semver patch](https://github.com/gnolang/gno/pull/536/commits/725bbbb5501f4672e7f02ef2e51d9910527b1970)